lessons/lesson3.md

```diff
@@ -0,0 +1,151 @@
+## Lesson 3: Fire Event using Event SDK
+
+### Fire event
+Once you set up the app and register the event provider, now you can make user click like button as fire event by adding code to your project firefly app
+
+Within the newly created app, Firstly, set up `package.json` with the lists of dependencies, version, etc. 
+Then `manifest.yml` lists the declaration of serverless actions including name, source files, runtime kind, default params, annotations, and so on
+Note: here put in the `providerId` and `eventCode`from lesson 2 and `orgId`. `apiKey`, `accessToken`from console integration from lesson 1
+
+Below is a sample `manifest.yml` 
+```javascript
+packages:
+  __APP_PACKAGE__:
+    license: Apache-2.0
+    actions:
+      event:
+        function: actions/event/index.js
+        web: 'yes'
+        runtime: 'nodejs:10'
+        inputs:
+          LOG_LEVEL: debug
+          orgId: <YOUR-ORG-ID>
+          apiKey:  <YOUR-apiKey>
+          accessToken: <YOUR-accessToken>
+          providerId: <YOUR-providerId>
+          eventCode: <YOUR-eventCode>
+        annotations:
+          final: true
+```
+The action to fire event is called `event` here, currently you app only has on action `event` :
+
+* Source code is at `actions/event/index.js`
+* It is a [web action](https://github.com/AdobeDocs/adobeio-runtime/blob/master/guides/creating_actions.md#invoking-web-actions)
+* The action will be run in the `nodejs:10` [runtime container on I/O Runtime](https://github.com/AdobeDocs/adobeio-runtime/blob/master/reference/runtimes.md)
+* It has some [default params](https://github.com/AdobeDocs/adobeio-runtime/blob/master/guides/creating_actions.md#working-with-parameters) such as `LOG_LEVEL`, `orgId`, `apiKey`, which are automatically available in the `params` object of the action without passing it to the action for every invocation. The `final` annotation set as `true` tells that those params are immutable.
+
+Now let's have a deeper look at the action's source code.
+also find source code[here](https://github.com/AdobeDocs/adobeio-codelab-customevent-demo/blob/master/actions/event/index.js)
+
+```javascript
+/**
+ * This action when user click the like button, fire an event
+ */
+
+const { Core } = require('@adobe/aio-sdk')
+const fetch = require('node-fetch')
+const ZonedDateTime = require('@js-joda/core').ZonedDateTime
```

Comment:
why js-joda and not OOTB js Date object?

Reply:
This sample code is from Sangeetha, I need to check with her